docs/authorization/access-policies-guide.md

```diff
@@ -0,0 +1,312 @@
+# About DataHub Access Policies
+
+<FeatureAvailability/>
+
+At the lowest level of DataHub's access control model is the concept of **Access Policies**. In conjunction with [Roles](./roles.md), Access Policies determine what users are allowed to do on DataHub.
+More specifically, Access Policies define *who* can *do what* to *which resources*.
+
+A few policies in plain English include
+
+- Dataset Owners should be allowed to edit documentation, but not Tags.
+- Jenny, our Data Steward, should be allowed to edit Tags for any Dashboard, but no other metadata.
+- James, a Data Analyst, should be allowed to edit the Links for a specific Data Pipeline he is a downstream consumer of.
+- The Data Platform team should be allowed to manage users & groups, view platform analytics, & manage policies themselves.
+
+Each of these can be implemented by constructing DataHub Access Policies.
+
+In this guide, we'll describe how to create & manage Access Policies.
+
+## Access Policies Setup, Prerequisites, and Permissions
+
+What you need to manage Access Policies on DataHub:
+
+* **Manage Policies** privilege
+
+This platform privilege allows users to create, edit, and remove all Access Policies on DataHub. Therefore, it should only be
+given to those users who will be serving as Admins of the platform. The default `Admin` role has this privilege.
+
+
+## Using Access Policies
+
+### Creating a new Policy
```

Comment:
Consider splitting this section into two -- `Creating a Metadata Policy` and `Creating a Platform Policy`; the current set of steps are hard to follow because there are nuances for each policy type & it's hard to know what is relevant to each use case